demos/serverless/deploy.js

```diff
@@ -20,8 +41,11 @@ function usage() {
   console.log(`  -a, --application                    Browser application to deploy, default '${app}'`);
   console.log(`  -e, --event-bridge                   Enable EventBridge integration, default is no integration`);
   console.log(`  -c, --chime-endpoint                 AWS SDK Chime endpoint, default is '${chimeEndpoint}'`);
+  console.log(`  -p, --service-principal              Service principal for meeting related resources, default is '${chimeServicePrincipal}'`)
```

Comment:
This is just for capturing right? If so, should we say so? Meeting related resources seem to include other things too.

Reply:
This would be used anywhere in the SAM template that currently points to the `chime.amazonaws.com` service principal.  It is used for testing.